recipes/cabinetry/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "cabinetry" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 88accfd0a6138e1b0c7ce0b5f44a746a53ebed8488bea02a5333846fee71f30a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - cabinetry = cabinetry.cli:cabinetry
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - pyhf[minuit] ~=0.6.3
```

This extras syntax is pip-specific and conda doesn't have a concept of `extras`:
```suggestion
    - pyhf ~=0.6.3
```

---

Ah thanks @chrisburr. Does Conda-forge understand compatible release syntax (`~=`)?

---

This is a `conda` question rather than a `conda-forge` one as this is resolved at install time on clients. Recent-ish versions of `conda` do but older ones don't. (IIRC we patch it with repodata patches to make older clients work.)